site2/docs/txn-use.md

```diff
@@ -20,21 +20,29 @@ This section provides an example of how to use the transaction API to send and r
 
 2. Enable transaction. 
 
-    Change the configuration in the `broker.conf` file.
+    Change the configuration in the `broker.conf` or `standalone.conf` file.
 
     ```
+    //mandatory configuration, used to enable transaction coordinator
     transactionCoordinatorEnabled=true
+   
+    //mandtory configuration, used to create systemTopic used for transaction buffer snapshot
     systemTopicEnabled=true
+   
     ```
 
-    If you want to enable batch messages in transactions, follow the steps below.
+    * If you want to enable batch messages in transactions, follow the steps below.
 
     Set `acknowledgmentAtBatchIndexLevelEnabled` to `true` in the `broker.conf` or `standalone.conf` file.
 
       ```
       acknowledgmentAtBatchIndexLevelEnabled=true
       ```
 
+    * [message deduplication](cookbooks-deduplication.md) is a mandatory configuration, if need to guarantee exactly-once semantics.
+    You can enable message deduplication at the broker level, the namespace level, or the topic level according to your needs.
```

Comment:
the namespace level or the topic level according to your needs.

Reply:
@momo-jun Please help review this, thanks.